docs/learn/learn-xcm-pallet.md

```diff
@@ -10,12 +10,90 @@ slug: ../learn-xcm-pallet
 The XCM pallet ([`pallet-xcm`](https://github.com/paritytech/polkadot/tree/master/xcm/pallet-xcm))
 provides a developer-friendly interface for most of the common XCM messages.
 
-This pallet provides default implementations for many traits required by `XcmConfig`, which is
-highly useful for configuration.
+This pallet provides some default implementations for some traits required by `XcmConfig`, as well
+as an instance of the XCM Executor provided as a trait within the pallet's own configuration.
+`pallet-xcm` provides a default interface in the form of a pallet, that can manage and deal with
+XCM-related storage and higher-level dispatchable functions.
```

Comment:
What is this talking about? Default interface between which two systems? And what kind of XCM-related storage?

Reply:
It was meant to show that the pallet is used to provide a common/default interface for developers to interact with XCM, but perhaps this wording is incorrect. 

> And what kind of XCM-related storage?

Storage related to versioning, i.e `SafeXcmVersion`.

Reply:
Would it be more accurate to assume that the pallet is a 'wrapper' for the `xcm-executor`?

Reply:
It should say that it is an interface between FRAME and XCM. I would not even say that it is the default, because the transport pallets all require specifying an XCM executor.